<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="map_name" default="closerfinalmap"/>

    <node pkg="ros_waypoint_generator"
          type="ros_waypoint_generator"
          name="ros_waypoint_generator_node">
    </node>

    <node pkg="barista_gazebo" type="save_waypoint.py" name="save_waypoint_node" output="screen">
        <param name="way_points_file_path"
                type="string"
                value="$(find barista_gazebo)/waypoints_saved/$(arg map_name).pickle" />
    </node>


</launch>